scss/atoms/typography/_typography.scss

```diff
@@ -12,11 +12,11 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( min-width: $viewport-xs ) {
```

Comment:
@sfrisk , I merely took the font sizes @MichaelArestad had applied to different screen sizes and did them mobile first.
So you mean we should have : 
1. default:14px;
2. wider than 768px: 16px
3. wider than 920px: 20px
??
that would meen a font size of 14px for the range 400-768 and 20 for 720-968